### PR TITLE
Initial proposal for static holding site

### PR DESCRIPTION
**Pull request by hamishcampbell**
Provides a responsive landing page with some basic information for the 2025 conference.

Will need mailing list information to wire up the "stay in touch" form.

Screenshot 2024-09-19 at 11 56 38 PM

Screenshot 2024-09-19 at 11 57 10 PM

Screenshot 2024-09-19 at 11 57 25 PM

### CODE REVIEW

**Review by nf-s — APPROVED** [review]
Looks great to me!

File: holding-page-static/index.html [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+			<!-- One -->
+				<section id="one" class="spotlight style1 bottom">
+					<span class="image fit main"><img src="images/auckland.jpg" alt="Auckland City" /></span>
+					<div class="content">
+						<div class="container">
+							<div class="row">
+								<div class="col-4 col-12-medium">
+									<header>
+										<h2>Welcome to the South Pacific</h2>
+										<p>OSGeo Oceanaia invites the FOSS4G community to Auckland / Tāmaki Makaurau,
```
Comment (nf-s) [inline holding-page-static/index.html]:
```suggestion
										<p>OSGeo Oceania invites the FOSS4G community to Auckland / Tāmaki Makaurau, 
```

File: holding-page-static/index.html [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+				<section id="one" class="spotlight style1 bottom">
+					<span class="image fit main"><img src="images/auckland.jpg" alt="Auckland City" /></span>
+					<div class="content">
+						<div class="container">
+							<div class="row">
+								<div class="col-4 col-12-medium">
+									<header>
+										<h2>Welcome to the South Pacific</h2>
+										<p>OSGeo Oceanaia invites the FOSS4G community to Auckland / Tāmaki Makaurau, 
+											the City of Sales, largest city of New Zealand Aotearoa for FOSS4G 2025.
```
Comment (nf-s) [inline holding-page-static/index.html]:
```suggestion
											the City of Sails, largest city of New Zealand Aotearoa for FOSS4G 2025.
```

File: holding-page-static/index.html [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+					<span class="image fit main"><img src="images/aut.jpg" alt="AUT City Campus" /></span>
+					<div class="content">
+						<header>
+							<h2>Our Venue</h2>
+							<p>FOSS4G 2025 is proudly supported by Auckland University 
+								of Technology's School of Future Environments</p>
+							
+						</header>
+						<p>Located in the heart of Auckland City, the AUT City Campus is within minutes of
+							the central business district, has excellent public transport and accomodation options.
```
Comment (nf-s) [inline holding-page-static/index.html]:
```suggestion
							the central business district, has excellent public transport and accommodation options.
```

File: holding-page-static/index.html [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+						-->
+					</div>
+				</section>
+
+			<!-- Five -->
+				<section id="five" class="wrapper style2 special fade">
+					<div class="container">
+						<header>
+							<h2>Stay in touch</h2>
+							<p>Sign up to our newsletter to recieve updates and information about FOSS4G 2025</p>
```
Comment (nf-s) [inline holding-page-static/index.html]:
```suggestion
							<p>Sign up to our newsletter to receive updates and information about FOSS4G 2025</p>
```

**Review by nf-s — CHANGES_REQUESTED** [review]
Just noticed some small misspellings

**Comment by hamishcampbell** [comment]
> Just noticed some small misspellings

Legend, thank you!

**Review by nf-s — COMMENTED** [review]
Just tested it on my phone - it would be nice to center the "save the date" image. As it zooms into the side on smaller screens
IMG_6568

**Comment by hamishcampbell** [comment]
@nf-s good spotting, should be better now

IMG_9036

**Pull request merged**